lib/utils/rand.go

```diff
@@ -25,14 +25,16 @@ import (
 	"github.com/gravitational/trace"
 )
 
-// CryptoRandomHex returns hex encoded random string generated with crypto-strong
-// pseudo random generator of the given bytes
-func CryptoRandomHex(len int) (string, error) {
-	randomBytes := make([]byte, len)
```

Comment:
Nit: It's not a big deal here since the function is small but in general descriptive variable names are preferred over single-letter variables like `b` or `l`. Makes the code much easier to read and follow.

Reply:
I'll change it to length. Shadowing len is what caught my eye.